.github/workflows/cross-post.yml

```
@@ -0,0 +1,95 @@
+      - name: Run unified cross-poster
+        if: ${{ github.event.inputs.platforms == 'all' || github.event.inputs.platforms == '' }}
+        env:
+          HASHNODE_API_TOKEN: ${{ secrets.HASHNODE_API_TOKEN }}
+          HASHNODE_PUBLICATION_ID: ${{ secrets.HASHNODE_PUBLICATION_ID }}
+        run: |
+          echo "Running unified cross-poster..."
+          tsx scripts/cross-post.ts \
+            "${{ github.event.inputs.post_slug || 'auto' }}" \
+            "${{ github.event.inputs.canonical_url || 'https://mac-portfolio.vercel.app' }}" \
+            "${{ github.event.inputs.publish_status || 'draft' }}"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)